From: Guillaume Delacour <gui@iroqwa.org>
Subject: Fix spelling errors in some modules.
Forwarded: http://www.inspircd.org/bugtrack/view.php?id=183
Last-Update: 2011-11-14

--- a/src/commands/cmd_server.cpp
+++ b/src/commands/cmd_server.cpp
@@ -41,7 +41,7 @@
 	}
 	else
 	{
-		user->WriteNumeric(ERR_NOTREGISTERED, "%s :You may not register as a server (servers have seperate ports from clients, change your config)",name.c_str());
+		user->WriteNumeric(ERR_NOTREGISTERED, "%s :You may not register as a server (servers have separate ports from clients, change your config)",name.c_str());
 	}
 	return CMD_FAILURE;
 }
--- a/src/modules/m_xline_db.cpp
+++ b/src/modules/m_xline_db.cpp
@@ -239,7 +239,7 @@
 
 	virtual Version GetVersion()
 	{
-		return Version("Keeps a dynamic log of all XLines created, and stores them in a seperate conf file (xline.db).", VF_VENDOR);
+		return Version("Keeps a dynamic log of all XLines created, and stores them in a separate conf file (xline.db).", VF_VENDOR);
 	}
 };
 
--- a/src/modules/m_swhois.cpp
+++ b/src/modules/m_swhois.cpp
@@ -121,7 +121,7 @@
 
 	Version GetVersion()
 	{
-		return Version("Provides the SWHOIS command which allows setting of arbitary WHOIS lines", VF_OPTCOMMON | VF_VENDOR);
+		return Version("Provides the SWHOIS command which allows setting of arbitrary WHOIS lines", VF_OPTCOMMON | VF_VENDOR);
 	}
 };
 
--- a/src/modules/m_sqlauth.cpp
+++ b/src/modules/m_sqlauth.cpp
@@ -156,7 +156,7 @@
 
 	Version GetVersion()
 	{
-		return Version("Allow/Deny connections based upon an arbitary SQL table", VF_VENDOR);
+		return Version("Allow/Deny connections based upon an arbitrary SQL table", VF_VENDOR);
 	}
 };
 
--- a/src/modules/m_spanningtree/fjoin.cpp
+++ b/src/modules/m_spanningtree/fjoin.cpp
@@ -71,14 +71,14 @@
 	if (created)
 	{
 		chan = new Channel(channel, TS);
-		ServerInstance->SNO->WriteToSnoMask('d', "Creation FJOIN recieved for %s, timestamp: %lu", chan->name.c_str(), (unsigned long)TS);
+		ServerInstance->SNO->WriteToSnoMask('d', "Creation FJOIN received for %s, timestamp: %lu", chan->name.c_str(), (unsigned long)TS);
 	}
 	else
 	{
 		time_t ourTS = chan->age;
 
 		if (TS != ourTS)
-			ServerInstance->SNO->WriteToSnoMask('d', "Merge FJOIN recieved for %s, ourTS: %lu, TS: %lu, difference: %lu",
+			ServerInstance->SNO->WriteToSnoMask('d', "Merge FJOIN received for %s, ourTS: %lu, TS: %lu, difference: %lu",
 				chan->name.c_str(), (unsigned long)ourTS, (unsigned long)TS, (unsigned long)(ourTS - TS));
 		/* If our TS is less than theirs, we dont accept their modes */
 		if (ourTS < TS)
--- a/src/modules/m_delayjoin.cpp
+++ b/src/modules/m_delayjoin.cpp
@@ -79,7 +79,7 @@
 
 Version ModuleDelayJoin::GetVersion()
 {
-	return Version("Allows for delay-join channels (+D) where users dont appear to join until they speak", VF_VENDOR);
+	return Version("Allows for delay-join channels (+D) where users don't appear to join until they speak", VF_VENDOR);
 }
 
 void ModuleDelayJoin::OnNamesListItem(User* issuer, Membership* memb, std::string &prefixes, std::string &nick)
