From: Guillaume Delacour <gui@iroqwa.org>
Subject: Use File::Temp::tmpnam() instead of deprecated POSIX::tmpnam()
 File::Temp::tmpnam() is deprecated in perl >= 5.22
Origin: https://github.com/inspircd/inspircd/pull/1080
Bug-Debian: https://bugs.debian.org/809008
Last-Update: 2015-12-28

Index: inspircd/make/utilities.pm
===================================================================
--- inspircd.orig/make/utilities.pm
+++ inspircd/make/utilities.pm
@@ -28,7 +28,7 @@ use strict;
 use warnings FATAL => qw(all);
 
 use Exporter 'import';
-use POSIX;
+use File::Temp ();
 use Getopt::Long;
 use Fcntl;
 our @EXPORT = qw(make_rpath pkgconfig_get_include_dirs pkgconfig_get_lib_dirs pkgconfig_check_version translate_functions promptstring);
@@ -404,7 +404,7 @@ sub translate_functions($$)
 			my $tmpfile;
 			do
 			{
-				$tmpfile = tmpnam();
+				$tmpfile = File::Temp::tmpnam();
 			} until sysopen(TF, $tmpfile, O_RDWR|O_CREAT|O_EXCL|O_NOFOLLOW, 0700);
 			print "(Created and executed \e[1;32m$tmpfile\e[0m)\n";
 			print TF $1;
